docs/profile-definition-language.md

```diff
@@ -0,0 +1,161 @@
+# Benchmark Profile Definition Language
+<a name="BenchmarkProfileDefinitionLanguage"></a>
+The Profile Definition is a text file that describes an action sequence. Possible actions: sending a web request, memorizing data from the response as a variable, waiting for a couple of seconds.
+ 
+At the beginnig of the line there is a *control character or word* that defines the purpose of that line. Empty lines and lines starting with an unrecognized control word will be skipped.
+
+### Comment
+If the line starts with a semicolon character (;) the line will be skipped.
+```text
+; PROFILE DESCRIPTION: Small content editor task
+```
+There is only line comment, it is not possible to write an inline comment.
+
+### PATHSET
```

Comment:
I suggest we use the same name for PathSet everywhere in the documentation (pathset vs path set vs PathSet vs pathSet). We should use either one.